docs/extras/modules/data_connection/vectorstores/integrations/azuresearch.ipynb

```diff
@@ -211,15 +211,372 @@
+      "\n",
+      "Tonight, I’d like to honor someone who has dedicated his life to serve this country: Justice Stephen Breyer—an Army veteran, Constitutional scholar, and retiring Justice of the United States Supreme Court. Justice Breyer, thank you for your service. \n",
+      "\n",
+      "One of the most serious constitutional responsibilities a President has is nominating someone to serve on the United States Supreme Court. \n",
+      "\n",
+      "And I did that 4 days ago, when I nominated Circuit Court of Appeals Judge Ketanji Brown Jackson. One of our nation’s top legal minds, who will continue Justice Breyer’s legacy of excellence.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Perform a hybrid search\n",
+    "docs = vector_store.hybrid_search(\n",
     "    query=\"What did the president say about Ketanji Brown Jackson\", k=3\n",
     ")\n",
     "print(docs[0].page_content)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Create a new index with custom filterable fields "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.search.documents.indexes.models import (\n",
+    "    SearchableField,\n",
+    "    SearchField,\n",
+    "    SearchFieldDataType,\n",
+    "    SimpleField,\n",
+    "    ScoringProfile,\n",
+    "    TextWeights,\n",
+    ")\n",
+    "\n",
+    "embeddings: OpenAIEmbeddings = OpenAIEmbeddings(model=model, chunk_size=1)\n",
+    "embedding_function = embeddings.embed_query\n",
+    "\n",
+    "fields = [\n",
+    "    SimpleField(\n",
+    "        name=\"id\",\n",
+    "        type=SearchFieldDataType.String,\n",
+    "        key=True,\n",
+    "        filterable=True,\n",
+    "    ),\n",
+    "    SearchableField(\n",
+    "        name=\"content\",\n",
+    "        type=SearchFieldDataType.String,\n",
+    "        searchable=True,\n",
+    "        retrievable=True,\n",
+    "    ),\n",
+    "    SearchField(\n",
+    "        name=\"content_vector\",\n",
+    "        type=SearchFieldDataType.Collection(SearchFieldDataType.Single),\n",
+    "        searchable=True,\n",
+    "        dimensions=len(embedding_function(\"Text\")),\n",
+    "        vector_search_configuration=\"default\",\n",
+    "    ),\n",
+    "    SearchableField(\n",
+    "        name=\"metadata\",\n",
+    "        type=SearchFieldDataType.String,\n",
+    "        searchable=True,\n",
+    "        retrievable=True,\n",
+    "    ),\n",
+    "    # Additional field to store the title\n",
+    "    SearchableField(\n",
+    "        name=\"title\",\n",
+    "        type=SearchFieldDataType.String,\n",
+    "        searchable=True,\n",
+    "        retrievable=True,\n",
+    "    ),\n",
+    "    # Additional field for filtering on document source\n",
+    "    SearchableField(\n",
```

Comment:
Why make the source field searchable? Do we really want the source name to play a role when retrieving content? Maybe it should be just filterable?